languages/ruby/exercises/concept/conditionals-ternary/.docs/introduction.md

```diff
@@ -0,0 +1,21 @@
+A ternary conditional in Ruby is a shorter way of writing simple `if/else` statements. If an `if/else` statement contains only two branches, one for when the condition is true and one for when it is false, it can be re-written as a ternary conditional.
+
+It uses a combination of the `?` and `:` symbols, often called the ternary operator(s).
+
+For example:
+
+```ruby
+if traffic_light == 'green'
+  cross_the_road
+else
+  wait
+end
+```
+
+can be re-written as:
+
+```ruby
+traffic_light == 'green' ? cross_the_road : wait
+```
```

Comment:
:+1: great!

Reply:
Yes, excellent introduction and example!